.github/workflows/release.yml

```diff
@@ -90,6 +90,27 @@ jobs:
           files: dist/scie-pants*
           fail_on_unmatched_files: true
           discussion_category_name: Announcements
+      - name: Post to a Pantsbuild Slack '#announce'
+        id: slack
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: "C18RRR4JK"
+          # N.B.: You can muck with the JSON blob and see the results rendered here:
+          #  https://app.slack.com/block-kit-builder
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
```

Comment:
> "mrkdwn",

not a typo?

Reply:
Not a typo. You can follow the link I pasted if you dare for proof.

Reply:
I may be crazy, but that got to be another level.. almost military style abbreviation :P (at least in Sweden, they've shortened "grodman" to "grdman" so you have to spell it out rather than pronounce it ((literally "frogman" i.e. a diver)) )

Reply:
Coders are just plain super weird. The strength of opinions about truly irrelevant things is stunning. I've definitely worked with ppl who lk 2 abbr things pretty militantly before. Some of them clearly work for Slack.

Reply:
Go is opinionated about this: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

"The basic rule: the further from its declaration that a name is used, the more descriptive the name must be."

So now refactoring may require names to change to meet some arbitrary name length vs declaration distance metric...

Reply:
🤯

Reply:
> Variable names in Go should be short rather than long. This is especially true for local variables with limited scope. Prefer c to lineCount

Yucky...

Reply:
> So now refactoring may require names to change to meet some arbitrary name length vs declaration distance metric...

Not sure there's tools that will enforce this:

> These numeric guidelines are not strict rules. Apply judgement based on context, [clarity](https://google.github.io/styleguide/go/guide#clarity), and [concision](https://google.github.io/styleguide/go/guide#concision).

Reply:
So much of Go is the software equivalent of Rob Pike yelling at kids to get off his damn lawn.

Reply:
But I mostly empathize. That guideline comes from his well-founded allergy to JVM names like AdapterPrototypeBuilderSingletonFactory.

Reply:
guess there'll be a language for every conceivable style preference out there..

Reply:
As I ~said - we - as a class - are horrible people expending precious time and brain on just useless stuff. We whine about totally dumb things.

Reply:
it's a vent, to take a break from solving mind-consuming tasks :P